<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterUser</class>
 <widget class="QWidget" name="RegisterUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>321</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>User name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="rPasswordLabel">
      <property name="text">
       <string>Re-enter Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="colorLabel">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="colorComboBox">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwordEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="rPasswordEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
